lib/fluent/plugin/out_http.rb

```diff
@@ -20,6 +20,8 @@
 require 'fluent/tls'
 require 'fluent/plugin/output'
 require 'fluent/plugin_helper/socket'
+require 'aws-sigv4'
+require 'aws-sdk-core'
```

Comment:
These gems aren't always required. So, as @ashie mentioned, we need to verify whether `:aws_sigv4` method is used or not in #configure method.
When the aws related gems are not installed to attempt to use this feature, then the error should be displayed.

Reply:
Resolved.